wiki/Welcome/vi.md

```diff
@@ -0,0 +1,47 @@
+# Chào mừng
+
+Chào mừng đến osu!, một trò chơi nhịp điệu được phát triển bởi peppy với 4 chế độ chơi: osu!, chế độ bấm vòng tròn; osu!taiko, chế độ đánh trống; osu!catch, chế độ hứng quả; và osu!mania, chế độ .
```

Comment:
```suggestion
Chào mừng đến osu!, một trò chơi nhịp điệu được phát triển bởi peppy với 4 chế độ chơi: osu!, bấm vòng tròn theo nhịp điệu; osu!taiko, giả lập đánh trống; osu!catch, chế độ hứng salad hoa quả; và osu!mania, chế độ giã phím đập đàn.
```
I think you can just use anything you like to make it *interesting*
I've check for references in different languages, FR defines taiko mode as horizontal drum, and mania mode as a VSRG game.
So just make it so you feel them **lit**. So I think going with `chế độ đập đàn` can work.